HelloDotNet/Hello.cs

```diff
@@ -26,16 +25,11 @@ static void Main(string[] args)
                 Environment.Exit(1);
             }
 
-            LogManager.Adapter = new ConsoleAdapter(LogLevel.Info);
-            // ConsoleAdapter is a simple Common.Logging implementation that's provided by LaunchDarkly;
-            // Common.Logging has an equivalent class, but not on every platform
+            var ldConfig = Configuration.Default(SdkKey);
 
-            Configuration ldConfig = LaunchDarkly.Client.Configuration
-                .Default(SdkKey);
+            var client = new LdClient(ldConfig);
 
-            LdClient client = new LdClient(ldConfig);

```

Comment:
Here I've just changed explicit type declarations to `var`— nothing to do with the new SDK, just good C# practice.